src/pages/workspace/NewWorkspacePage.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import WorkspaceDefaultAvatar from '../../../assets/images/workspace-default-avatar.svg';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import Button from '../../components/Button';
+import AttachmentPicker from '../../components/AttachmentPicker';
+import Icon from '../../components/Icon';
+import {DownArrow, Upload} from '../../components/Icon/Expensicons';
+import CreateMenu from '../../components/CreateMenu';
+import Switch from '../../components/Switch';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+class NewWorkspacePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isEditPhotoMenuVisible: false,
+            name: '',
+            requestCall: false,
+        };
+
+        this.createMenuItems = this.createMenuItems.bind(this);
+    }
+
+    /**
+     * Create menu items list for avatar menu
+     *
+     * @param {Function} openPicker
+     * @returns {Array}
+     */
+    createMenuItems(openPicker) {
```

Comment:
NAB, these options really only need to be updated if the translation theme changes, but I don't think it's worth optimizing for so this function works for me.

Reply:
I left the function because we'll need it when we connect this with the backend, and we need to decide whether to show the `Delete Image` item or not based on the state.